process/eventsHandler.go

```diff
@@ -67,12 +67,17 @@ func checkArgs(args ArgsEventsHandler) error {
 
 // HandlePushEvents will handle push events received from observer
 func (eh *eventsHandler) HandlePushEvents(events data.BlockEvents) {
+	if events.Hash == "" {
```

Comment:
might worth setting these logs as `WARN` ? I think it shouldn't happen

Reply:
set the logs to WARN